docs/LOCALHOST.md

```diff
@@ -48,6 +49,12 @@ POST */_delete_by_query
 }
 ```
 
+Delete all indices:
+
+```none
+DELETE *
+```
+
```

Comment:
This and the readme in the OpenSearch docker directory have redundant instructions for clearing out the database. This is because it is easy to forget where the instructions are. If you are starting out for the first time, you're likely to look here. But if you're opening VS Code from `meadowlark-js` then you won't see this `docs` directory, and you'll be looking for the instructions somewhere else. Hence doubling up seemed useful in this circumstance.